### PR TITLE
SCSS login method and automation skeleton

### PR DESCRIPTION
**Pull request by alenaminsk**

### CODE REVIEW

File: src/test/java/appsCommon/PageInitializer.java [hunk 1]
```diff
@@ -355,6 +354,14 @@ public static void initializeAllPages() {
...
         nciNativeViewPage = new NCINativeViewPage();
 
+        /** SCSS Instance Variables */
+        owmVacancyPage = new OWMVacancyPage();
+        stadtmanVacancyPage = new StadtmanVacancyPage();
+        oWMVacancyStepsImplementation = new OWMVacancyStepsImplementation();
+        stadtmanVacancyStepsImplementation = new StadtmanVacancyStepsImplementation();
+        sCCSLoginStepsImplementation = new SCSSLoginStepsImplementation();
+
+
```
Comment (mzuhoor) [inline src/test/java/appsCommon/PageInitializer.java]: Extra space here

**Review by georgeb79 — APPROVED** [review]

**Review by mzuhoor — APPROVED** [review]

File: src/test/java/ServiceNow/SCSS/Steps/HooksSteps.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            scenarioNameForFolderCreation = s.getName() + File.separatorChar + exampleName;
+        }
+
+        if (!StringUtils.isEmpty(buildNumber)) {
+            scenarioNameForFolderCreation = buildNumber + File.separatorChar + scenarioNameForFolderCreation;
+        } else {
+            // create artifacts in new folder with current time-stamp
+            scenarioNameForFolderCreation = DateUtils.getFormatedDate("MM-dd-yyyy") + File.separatorChar
+                    + scenarioNameForFolderCreation;
+        }
```
[the remark this answers is not in the document]
Comment (alenaminsk) [reply src/test/java/ServiceNow/SCSS/Steps/HooksSteps.java:58]: added space

File: src/test/java/ServiceNow/SCSS/Steps/OWMVacancySteps.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+import appsCommon.PageInitializer;
+import com.nci.automation.web.CommonUtils;
+import io.cucumber.java.en.Given;
+
+public class OWMVacancySteps  extends PageInitializer {
+    @Given("User is on SCSS Landing page and user is {string}")
+    public void user_is_on_scss_landing_page_and_user_is(String user) {
+        sCCSLoginStepsImplementation.sCSSLogin(user);
+    }
```
[the remark this answers is not in the document]
Comment (alenaminsk) [reply src/test/java/ServiceNow/SCSS/Steps/OWMVacancySteps.java:12]: thank you, added

**Review by sarwarahmed1 — APPROVED** [review]

File: src/test/java/appsCommon/PageInitializer.java (hunk 1 above)
Comment (alenaminsk) [reply src/test/java/appsCommon/PageInitializer.java]: removed

File: src/test/java/ServiceNow/SCSS/Features/StadtmanVacancy.feature
Comment (zamant2) [inline src/test/java/ServiceNow/SCSS/Features/StadtmanVacancy.feature:1]: Any reason to push empty feature file on master

File: src/test/java/appsCommon/PageInitializer.java [hunk 4]
```diff
@@ -92,21 +92,17 @@
...
-import ServiceNow.NERD.Pages.NERDHomePage;
-import ServiceNow.NERD.Pages.NERDKnowledgebasePage;
-import ServiceNow.NERD.Pages.NERDOGCRAddNewEntryPage;
-import ServiceNow.NERD.Pages.NERDSubmissionsPage;
-import ServiceNow.NERD.Pages.NativeViewImpersonateUserPage;
-import ServiceNow.NERD.Pages.NativeViewMembersOfCongressPage;
+import ServiceNow.NERD.Pages.*;
 import ServiceNow.NERD.StepsImplementation.NERDLoginStepsImplementation;
 import ServiceNow.NERD.StepsImplementation.NativeViewImpersonateUser;
+<<<<<<< HEAD
```
Comment (zamant2) [inline src/test/java/appsCommon/PageInitializer.java:98]: please remove this

File: src/test/java/appsCommon/PageInitializer.java [hunk 5]
```diff
@@ -116,6 +112,7 @@
 import ServiceNow.SEER.Pages.SEERIncidenceDatabaseDetailsPage;
 import ServiceNow.SEER.Pages.SEERLandingPage;
 import ServiceNow.SEER.Pages.SEERUserRegistrationPage;
+>>>>>>> 2e07fd64e242f85a5a8e43ce9cea5d684402a3eb
```
Comment (zamant2) [inline src/test/java/appsCommon/PageInitializer.java:115]: Remove this line please

File: src/test/java/ServiceNow/SCSS/Features/StadtmanVacancy.feature
Comment (alenaminsk) [reply src/test/java/ServiceNow/SCSS/Features/StadtmanVacancy.feature:1]: removed

File: src/test/java/appsCommon/PageInitializer.java (hunk 4 above)
Comment (alenaminsk) [reply src/test/java/appsCommon/PageInitializer.java:98]: removed

File: src/test/java/appsCommon/PageInitializer.java (hunk 5 above)
Comment (alenaminsk) [reply src/test/java/appsCommon/PageInitializer.java:115]: removed

File: src/test/java/appsCommon/PageInitializer.java [hunk 6]
```diff
@@ -92,21 +92,17 @@
...
+import ServiceNow.NERD.Pages.*;
 import ServiceNow.NERD.StepsImplementation.NERDLoginStepsImplementation;
 import ServiceNow.NERD.StepsImplementation.NativeViewImpersonateUser;
+<<<<<<< HEAD
+import ServiceNow.SCSS.Pages.OWMVacancyPage;
+import ServiceNow.SCSS.Pages.StadtmanVacancyPage;
+import ServiceNow.SCSS.StepsImplementation.OWMVacancyStepsImplementation;
+import ServiceNow.SCSS.StepsImplementation.SCSSLoginStepsImplementation;
+import ServiceNow.SCSS.StepsImplementation.StadtmanVacancyStepsImplementation;
+import ServiceNow.SEER.Pages.*;
```
[the remark this answers is not in the document]
Comment (alenaminsk) [reply src/test/java/appsCommon/PageInitializer.java:104]: That is the way Intellij compiles pages uniting imports with *
Nerd. Pages, Charms.pages, EIDP.pages are displayed the same way

**Review by zamant2 — APPROVED** [review]

**Pull request merged**